DXRFixes/DeusEx/Classes/ElectricityEmitter.uc

```diff
@@ -42,7 +43,20 @@ function CalcTrace(float deltaTime)
         // shock whatever gets in the beam
         if ((HitActor != None) && (lastDamageTime >= damageTime))
         {
-            HitActor.TakeDamage(damageAmount, Instigator, HitLocation, vect(0,0,0), 'Shocked');
+            damageAmt = damageAmount;
+
+            //Scale damage for the player, since they get a combat difficulty multiplier
+            //This helps with the Vandenberg computer room, the electricity in the Aquinas Substation
+            //and the Hong Kong Helibase (and many more)
+            if (HitActor.IsA('#var(PlayerPawn)')){
+#ifdef hx
+                damageAmt = Clamp(damageAmount / (HXGameInfo(Level.Game).CombatDifficulty),1,damageAmount);
```

Comment:
I don't think this folder gets compiled for HX, but also HX doesn't scale enviro damage with combat difficulty since it just uses vanilla rules where only Shot and Autoshot get scaled?